README.md

```diff
@@ -127,6 +129,7 @@ options:
   function reference. For the stage to be executed, the condition *must*
   return `true`. When the stage is skipped, the input is forwarded to
   the next step if there's one.
+* `:unless`: The same as the `:if` option, but act as having the opposite effect.
```

Comment:
Let's be more explicit here.

I'd rephrase this to not include the work "same":

`The opposite of the `:if`, the callback function *must* return `false` for the stage to be run.